docs/operate/reference/services/discovery/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Discovery Service"
+linkTitle: "Discovery"
+description: "The discovery service is used to discover resources on a machine."
+type: docs
+weight: 50
+no_list: true
+#icon: true
+#images: ["/services/icons/discovery.svg"]
+tags: ["discovery", "services"]
+no_service: true
+date: "2025-03-03"
+# updated: ""  # When the content was last entirely checked
+# SMEs: John N.
+modulescript: true
+---
+
+The discovery service is used to discover resources on a machine.
```

Comment:
passive voice

Reply:
Does the discovery service discover resources as in components OR does it discover attached hardware (and the paths under which the hardware is available) or both? Clarifying this further is I think important.